source/geod24/concurrency.d

```diff
@@ -1126,6 +1129,143 @@ private:
     size_t m_pos;
 }
 
+/// Test for the use `SchedulingTerminated`
+unittest
+{
+    import std.conv;
+
+    alias ChannelT = Channel!Message;
```

Comment:
`Message` is `private`, so it doesn't make sense that this unittest has `///`.

`///` means it will appear in the generated HTML documentation, and `//` means it will not.